metadata.yaml

```diff
@@ -6,6 +6,7 @@ description: |
   MinIO's high-performance object storage suite is software defined and
   enables customers to build cloud-native data infrastructure for
   machine learning, analytics and application data workloads.
+docs: https://discourse.charmhub.io/t/minio-docs-index/10861
```

Comment:
```suggestion
docs: https://discourse.charmhub.io/t/10861
```

Reply:
Great catch @beliaev-maksim

Reply:
@beliaev-maksim FYI I believe that once the topic number is included, Discourse will just ignore the "slug". But this way is cleaner anyway.